webapp/webpack.config.js

```diff
@@ -35,7 +35,9 @@ module.exports = {
                     {
                         loader: 'sass-loader',
                         options: {
-                            includePaths: ['node_modules/compass-mixins/lib', 'sass'],
+                            sassOptions: {
+                                includePaths: ['node_modules/compass-mixins/lib', 'sass'],
+                            },
```

Comment:
Same question here: Also include this into https://github.com/mattermost/mattermost-plugin-starter-template?

Reply:
This is dependant of the version of one of the libraries (not sure if webpack or babel). If we were to bump the dependencies of the plugin starter template (which I think we should), this will be needed.

Reply:
0/5 maybe just bump the version and include the change?